pkg/kdump/kdump-view.jsx

```diff
@@ -263,12 +265,43 @@ export class KdumpPage extends React.Component {
     }
 
     handleTestSettingsClick() {
+        // if we have multiple targets defined, the config is invalid
+        const target = this.props.kdumpStatus.target;
+        let verifyMessage;
+        if (!target.multipleTargets) {
+            let path = target.path || "/var/crash";
+            if (target.type === "local") {
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("Results of the crash will be stored in $0 as $1, if kdump is properly configured."),
+                    <span className="pf-v5-u-font-family-monospace-vf">{path}</span>,
+                    <span className="pf-v5-u-font-family-monospace-vf">vmcore</span>);
+            } else if (target.type === "ssh") {
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("Results of the crash will be copied through SSH to $0 as $1, if kdump is properly configured."),
+                    <span className="pf-v5-u-font-family-monospace-vf">{`${target.server}:${path}`}</span>,
+                    <span className="pf-v5-u-font-family-monospace-vf">vmcore</span>);
+            } else if (target.type === "nfs") {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19107-20230821-084635-b5861964-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)